pdns/pdns.conf-dist

```diff
@@ -462,7 +462,7 @@
 #################################
 # socket-dir	Where the controlsocket will live
```

Comment:
maybe write something here about how there is a silent default to /var/run if not chrooted - if that's not too much text.